tests/writer_util.py

```diff
@@ -36,7 +36,7 @@ def _scalar_p(ion_type, value, expected, force_stream_end):
     if force_stream_end:
         events += _STREAM_END_EVENT
     return WriterParameter(
-        desc='SCALAR %s - %s' % (ion_type.name, expected),
+        desc='SCALAR %s - %r - %r' % (ion_type.name, value, expected),
```

Comment:
This causes failure to render for raw byte sequences that cannot be encoded as UTF-8 making it impossible to get the actual failure from the report.